code/_compile_options.dm

```diff
@@ -26,19 +26,19 @@
 #endif
 
 //Update this whenever you need to take advantage of more recent byond features
-#define MIN_COMPILER_VERSION 513
-#define MIN_COMPILER_BUILD 1523
+#define MIN_COMPILER_VERSION 514
```

Comment:
Dont update minimum versions unless it is explicitly to make use of new features